bindings/go/whisper.go

```diff
@@ -99,9 +101,14 @@ var (
 
 // Allocates all memory needed for the model and loads the model from the given file.
 // Returns NULL on failure.
-func Whisper_init(path string) *Context {
+func Whisper_init(path string, flashAttn, disableLog bool) *Context {
```

Comment:
I'm not sure about adding these context parameters like this. Perhaps instead there should be a method that takes `ContextParams` as an argument similar to the [Java binding](https://github.com/ggml-org/whisper.cpp/blob/7fd6fa809749078aa00edf945e959c898f2bd1af/bindings/java/src/main/java/io/github/ggerganov/whispercpp/WhisperCpp.java#L46)?